test_start_helpers.sh

```diff
@@ -92,6 +92,25 @@ function get_ios_device_name {
   fi
 }
 
+
+function start_script_py {
```

Comment:
Should we split up start_script-methods so we have:

```
function install_and_start_script {
  install_libs
  start_script
}
```

Both for python and js. This code looks good but I am worried it's going to end up like the huge function for the js start_script when special cases starts piling up.